play/symphony-conductor/js/app.js

```diff
@@ -316,9 +339,9 @@ function createInstrumentElements() {
 
         // have tooltip show name on hover
         el.setAttribute("data-toggle", "tooltip");
-        el.setAttribute("title", instrument.description + 
-        `<br><a href="${instrument.link}" target="_blank">Click to learn more about this instrument</a>`); 
-        
+        el.setAttribute("title", instrument.description +
```

Comment:
You shouldn't use concatenation and a template string, just put ```instrument.description``` inside of the template string